docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -321,7 +321,7 @@ The JDBC lookups will poll a database to populate its local cache. If the `tsCol
 |`table`|The table which contains the key value pairs|Yes||
 |`keyColumn`|The column in `table` which contains the keys|Yes||
 |`valueColumn`|The column in `table` which contains the values|Yes||
-|`filter`|The filter to use when selecting lookups, this is is used to create a where clause on lookup population|No| No Used|
+|`filter`|The filter to use when selecting lookups, this is is used to create a where clause on lookup population|No| Not Used|
```

Comment:
1) "is is"
2) Extra space before "Not"
3) I don't think it has to be aligned tsColumn. The goal is to make it readable. I think "No filter" is more clear.